src/library/VTKExtensions/Readers/Testing/TestF3DMetaReader.cxx

```diff
@@ -0,0 +1,20 @@
+#include <vtkNew.h>
+#include <vtkTestUtilities.h>
+
+#include "F3DReaderInstantiator.h"
+#include "vtkF3DMetaReader.h"
+
+int TestF3DMetaReader(int argc, char* argv[])
+{
+  F3DReaderInstantiator ReaderInstantiator;
+
+  const char* fname = vtkTestUtilities::ExpandDataFileName(argc, argv, "data/cow.vtp");
```

Comment:
I always disliked this `vtkTestUtilities::ExpandDataFileName` function:
- It returns a `const char*` which has to be deleted manually
- The data path is relative (to `Testing/Data` I believe it's the reason why you need the symlink?)

What about creating a `F3D_TESTING_DATA_PATH` CMake variable, and pass it to the VTK modules with `target_compile_definitions()`?

Reply:
this is what I mean by custom system, but vtk should be improved at some point

Reply:
Ok, then having a custom system for now seems easy enough to do it.